lib/ui/pointer.dart

```diff
@@ -54,13 +54,26 @@ enum PointerDeviceKind {
   unknown
 }
 
+/// The kind of [PointerDeviceKind.signal].
+enum PointerSignalKind {
+  /// The event is not associated with a pointer signal.
+  none,
+
+  /// A pointer-generated scroll (e.g., mouse wheel or trackpad scroll).
+  scroll,
+
+  /// An unknown pointer signal kind.
+  unknown
+}
+
 /// Information about the state of a pointer.
 class PointerData {
```

Comment:
Any places you expand the API of dart:ui also need to updated in the corresponding "stub_ui" library with an Unsupported implementation - this is how we're testing dart2js and dartdevc

Reply:
Done.

Reply:
Thanks!